packages/core/src/external/cda/remove-b64.ts

```diff
@@ -1,13 +1,17 @@
 import { toArray } from "@metriport/shared";
-import { XMLBuilder } from "fast-xml-parser";
 import { createXMLParser } from "@metriport/shared/common/xml-parser";
+import { XMLBuilder } from "fast-xml-parser";
 import { BINARY_MIME_TYPES } from "../../util/mime";
 
 const notesTemplateId = "2.16.840.1.113883.10.20.22.2.65";
 const resultsTemplateId = "2.16.840.1.113883.10.20.22.2.3.1";
 const b64Representation = "B64";
 
-export function removeBase64PdfEntries(payloadRaw: string): string {
+export function removeBase64PdfEntries(payloadRaw: string): {
+  documentContents: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  b64Attachments: any[]; // TODO: improve typing
```

Comment:
yea, was going to say, loooots of `any` in the file above.

Reply:
not sure if this is something we want to address right away tbh

Reply:
there's no XML general type or something?